evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -649,353 +644,6 @@ def test_graph_binary_objective_vs_threshold(mock_cb_thresholds, data_type, X_y_
     assert np.array_equal(data['y'], mock_cb_thresholds.return_value['score'])
 
 
-def check_partial_dependence_dataframe(pipeline, part_dep, grid_size=20):
```

Comment:
Moved the partial_dependence tests over to their own file for organizational purposes.